builtin/credential/aws/cli.go

```diff
@@ -98,7 +98,7 @@ func (h *CLIHandler) Auth(c *api.Client, m map[string]string) (*api.Secret, erro
 		headerValue = ""
 	}
 
-	creds, err := RetrieveCreds(m["aws_access_key_id"], m["aws_secret_access_key"], m["aws_security_token"])
+	creds, err := RetrieveCreds(m["aws_access_key_id"], m["aws_secret_access_key"], m["aws_security_token"], hclog.Default())
```

Comment:
This does not derive any of the logging properties from the main command, so it might result in different format, log level, etc.

Reply:
How would we get the logging properties from the main command here? I haven't found a way to do that.

Reply:
The log would be from the client (result from a `vault login` command) for this particular case so it might be fine, though `hclog.Default()` logs up to `Info` only, so it would skip any `Debug` output.

Reply:
I'm not seeing anywhere else in the vault login command that verbosity levels are set, so in 2271d6c I added a `log_level` option to the aws cli login command, and used that to configure the logger passed to `RetrieveCreds()`. Or maybe it's preferable to just pass a null logger here?